README.md

```diff
@@ -145,6 +157,21 @@ Just like any other Blade component, check out the [Laravel Blade docs](https://
 </x-rapidez::accordion>
 ```
 
+#### Slideover
+```blade
+<label for="my-slideover">
+    Open Slideover
+</label>
+
+<x-rapidez::slideover id="my-slideover" title="Example Slideover">
+    Your slideover content goes here
+</x-rapidez::slideover>
+```
+Make sure to add this class to your body tag to prevent scrolling when the slideover is open:
+```html
+<body class="has-[.prevent-scroll:checked]:overflow-clip">
+```
```

Comment:
We do have this twice now:

- https://github.com/rapidez/blade-components#slideover
- https://github.com/rapidez/blade-components#slideover-1

Could you fix that @JimmyHoenderdaal?